web/src/service-worker/index.ts

```diff
@@ -11,8 +11,15 @@ const pendingLoads = new Map<string, AbortController>();
 // Create a unique cache name for this deployment
 const CACHE = `cache-${version}`;
 
+const APP_RESOURCES = [
```

Comment:
`build` only contains the app files in production builds. 
`files` will contain the contents of `/web/static` in both prod and dev builds.